sw/airborne/boards/mateksys/FC-H743-SLIM/mcuconf.h

```diff
@@ -0,0 +1,593 @@
+/*
+    ChibiOS - Copyright (C) 2006..2018 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#ifndef MCUCONF_H
+#define MCUCONF_H
+
+/*
+ * Enforce old versions of the chip
+ */
+#define STM32_ENFORCE_H7_REV_XY
```

Comment:
Are you sure they use old versions?

Reply:
I recall checking for that and ardupilot is also using this flag